packages/cdk-dasm/lib/dasm.ts

```diff
@@ -70,6 +89,25 @@ export async function dasmTypeScript(template: Template, options: DisassemblerOp
   code.openBlock(`constructor(scope: Construct, id: string, props: StackProps = {})`);
   code.line(`super(scope, id, props);`);
 
+  // Parameter definitions
+
+  for (const param of parameters) {
+
+    code.indent(`const ${param.id} = new CfnParameter(this, '${param.id}', {`);
+
+    // Default is needed, so if not supplied in CFN, then we'll
```

Comment:
Why is default needed?

Reply:
Hey @eladb ,

I couldn't get the stack to deploy or synth correctly without it:

```
MyStack: deploying...
MyStack: creating CloudFormation changeset...

 ❌  MyStack failed: ValidationError: Parameters: [MyParam] must have values
Parameters: [MyParam] must have values
```

Using these 2 commands:

```
cdk deploy -c MyParam=test
cdk deploy
```

And when using `cdk synth -c MyParam=test`, I don't see MyParam being set anywhere in the output yaml.

Is there another command I should be using to include parameters?